test/suite/Tools.test.js

```diff
@@ -39,4 +39,53 @@ describe( 'Tools test', () => {
 			} );
 		} );
 	} );
+
+	describe( 'Tools.bidiIsolate', () => {
+		const cases = [
+			{
```

Comment:
Just to make a point to our reviewers (and ourselves 😄 ) I'd add a test here that shows that any raw string with html in it is being outputted safely; something like `<em>foo</em> => &lt;em&gt;foo&lt;/em&gt;`

Reply:
Done.